main.cpp

```diff
@@ -207,289 +225,306 @@ Thing 2) Family
+}
+void InputSection::adjustLineLevel(float currentLineLevel, float desiredLevel)
+{
+    float amountChanged = desiredLevel - currentLineLevel;
+    std::cout << "Adjusted by " << amountChanged << "dB.";
+}
+float InputSection::pad(float currentLevel)
+{
+    return currentLevel - 30.0f;
+}
 
 struct MonitorSection 
 {
-    // selected cue output numbers
     int cueOutput = 4;
-    // selected master output numbers
     int masterOutput = 2;
-    // master level
     float masterLevelInDB = 55.6f;
-    // headphone level
     float headphoneLevelInDB = 12.5f;
-    // talkback level
     float talkbackLevelInDB = 20.4f;
 
-    //Route audio to different cues/playback outputs
     void route(int cueOutputToAdd, int masterOutputToAdd);
-    //Adjust master level
     float adjustMasterLevel(float currentMasterLevel, float amountToAdd);
-    //Adjust talkback level
     float adjustTalkbackLevel(float currentTalkbackLevel, float amountToAdd);
 };
-/*
-Thing 9) Aux Section
-5 properties:
-    1) aux send 1 level
-    2) aux send 2 level
-    3) aux send 3 level
-    4) aux send 4 level
-    5) aux send 5 level
-3 things it can do:
-    1) route audio to selected sends
-    2) adjust gain sent to each send
-    3) Group two mono sends as stereo
- */
+void MonitorSection::route(int cueOutputToAdd, int masterOutputToAdd)
+{
+    std::cout << "Routing to cue " << cueOutputToAdd << " and master output " << masterOutputToAdd;
+}
+float MonitorSection::adjustMasterLevel(float currentMasterLevel, float amountToAdd)
+{
+    return currentMasterLevel = amountToAdd;
+}
+float MonitorSection::adjustTalkbackLevel(float currentTalkbackLevel, float amountToAdd)
+{
+    return currentTalkbackLevel + amountToAdd;
+}
+
  struct AuxSection 
  {
-    // aux send 1 level
     float auxSend1Level = 20.f;
-    // aux send 2 level
     float auxSend2Level = 10.f;
-    // aux send 3 level
     float auxSend3Level = 44.5f;
-    // aux send 4 level
     float auxSend4Level = 14.f;
-    // aux send 5 level
     float auxSend5Level = 2.6f;
 
-    //Route audio to selected sends
     void routeToSend(bool send1, bool send2, bool send3, bool send4, bool send5);
-    //Adjust gain sent to each send
     void newGain(float send1Gain, float send2Gain, float send3Gain, float send4Gain, float send5Gain);
-    //Group two mono sends as stereo
     void createStereoSend(std::string leftAuxSend, std::string rightAuxSend);
  };
+void AuxSection::routeToSend(bool send1, bool send2, bool send3, bool send4, bool send5)
+{
+    if(send1)
+    {
+        std::cout << "Routed to send 1";
+    }
+    if(send2)
+    {
+        std::cout << "Routed to send 2";
+    }
+    if(send3)
+    {
+        std::cout << "Routed to send 3";
+    }
+    if(send4)
+    {
+        std::cout << "Routed to send 4";
+    }
+        if(send5)
+    {
+        std::cout << "Routed to send 5";
+    }
+}
+
+void AuxSection::newGain(float send1Gain, float send2Gain, float send3Gain, float send4Gain, float send5Gain)
```

Comment:
`newGain` is not an action. 
pick a better name that describes what this function does
example: `setSendGainLevels()`
- [ ] revise